core/arch/arm/encode.c

```diff
@@ -1098,7 +1100,8 @@ encode_opnd_ok(decode_info_t *di, byte optype, opnd_size_t size_temp, instr_t *i
     /* Roll back greedy reglist if necessary */
     if (di->reglist_stop > 0 && optype_is_reg(optype) &&
         (!di->reglist_simd || !optype_is_gpr(optype)) &&
-        di->reglist_stop - 1 > di->reglist_start && di->reglist_stop == opnum) {
+        di->reglist_stop - di->reglist_start > di->reglist_min_num &&
```

Comment:
For no minimum, previously we required stop-1>start == stop-start>1, but now you're requiring stop-start>0.  We don't support an empty list in our encoder, so >1 seems right.  Maybe that means reglist_min_num (and the min_num param) should be 1 instead of 0 everywhere?